module-tests/providers.tf

```diff
@@ -0,0 +1,19 @@
+provider "aws" {
+  region = var.region
+
+  profile = "bai-mgmt-gbl-identity-terraform"
+  assume_role {
+    role_arn = "arn:aws:iam::983558388183:role/bai-mgmt-gbl-sandbox-terraform"
```

Comment:
Ahh we definitely don't want this repo to be public anymore if we're going to put AWS account ids in it

Reply:
If we make this private. We need to overhaul the Workflows. Right now they are using reusable workflows from CP and I think they would run in the Github runners instead of our ARC runners.

Reply:
I removed the tests for now since Github didn't allow for forked repo to be converted to private

Reply:
Make sure to squash, or the values are going to be in the Git history anyway.